pkg/20.10.8/config.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+DOCKER_GIT_COMMIT="0e685c6ac0bddd7b2ba7bcaaeb519746ad249a29"
+
+TEST_OS_IMAGE_NAME=(oraclelinux centos ubuntu)
+TEST_OS_IMAGE_TAG[0]="7 8"
+TEST_OS_IMAGE_TAG[1]="centos7 centos8"
+TEST_OS_IMAGE_TAG[2]="16.04 18.04 20.04"
```

Comment:
If this docker version shouldn't be tested at ubuntu `16.04`, it shouldn't be added here.

```
-TEST_OS_IMAGE_TAG[2]="16.04 18.04 20.04"
+TEST_OS_IMAGE_TAG[2]="18.04 20.04"
```

Reply:
@rawmind0 Ah! right. let me change it right away.